spec/models/manageiq/providers/vmware/infra_manager/vm_spec.rb

```diff
@@ -5,7 +5,8 @@
   let(:host) { FactoryBot.create(:host_vmware_esx, :ext_management_system => ems) }
   let(:vm)   { FactoryBot.create(:vm_vmware, :ext_management_system => ems, :host => host) }
   let(:power_state_on)        { "poweredOn" }
-  let(:power_state_suspended) { "poweredOff" }
```

Comment:
This was odd, not sure why `power_state_suspended` was using `"poweredOff"` instead of `"suspended"` :shrug: